src/ApparentHorizons/StrahlkorperFunctions.hpp

```diff
@@ -23,6 +25,116 @@ struct not_null;
 /// \ingroup SurfacesGroup
 /// Contains functions that depend on a Strahlkorper but not on a metric.
 namespace StrahlkorperFunctions {
+/// @{
+/*!
+ * \f$(\theta,\phi)\f$ on the Strahlkorper surface.
+ * Doesn't depend on the shape of the surface.
+ */
+template <typename Fr>
+tnsr::i<DataVector, 2, ::Frame::Spherical<Fr>> theta_phi(
+    const ::Strahlkorper<Fr>& strahlkorper);
+
+/*!
+ * \f$(\theta,\phi)\f$ on the Strahlkorper surface.
+ * Doesn't depend on the shape of the surface.
+ */
+template <typename Fr>
+void theta_phi(
+    const gsl::not_null<tnsr::i<DataVector, 2, ::Frame::Spherical<Fr>>*>
+        theta_phi,
+    const ::Strahlkorper<Fr>& strahlkorper);
+/// @}
+
+/// @{
+/*!
+ * `r_hat(i)` is \f$\hat{r}^i = x_i/\sqrt{x^2+y^2+z^2}\f$ on the
```

Comment:
Is there a reason why you have chosen upper / lower indices for `theta_phi`, `rhat`, etc? It probably doesn't matter much, unless you want to use them with tensor expressions at some point. If there's a design decision behind this though, then please add some documentation, e.g. to a namespace or to these function dox.